sdk/identity/identity/review/identity.api.md

```diff
@@ -104,6 +104,16 @@ export interface AzureDeveloperCliCredentialOptions extends MultiTenantTokenCred
     tenantId?: string;
 }
 
+// @public (undocumented)
```

Comment:
We should document this and add a CHANGELOG entry - not blocking if you need a dev build and prefer to do that separately

Reply:
added